src/components/Talk/ParameterSlider.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div>
+    <div class="parameter-header">
+      <label :for="`parameter-${sliderKey}`" class="parameter-label">
+        {{ label }}
+      </label>
+      <input
+        :id="`parameter-${sliderKey}`"
+        :value="inputValueFixed"
+        class="parameter-input"
+        type="text"
+        @change="onUpdate"
+        @blur="onUpdate"
+      />
+    </div>
+    <BaseSlider
+      v-model="inputValue"
+      :min
+      :max
+      :step
+      :scrollStep
+      @valueCommit="$emit('update:modelValue', $event)"
+    />
+  </div>
+</template>
+
+<script setup lang="ts">
+import { computed, ref, watchEffect } from "vue";
+import BaseSlider from "@/components/Base/BaseSlider.vue";
+
+const props = defineProps<{
+  sliderKey: string;
+  min: number;
+  max: number;
+  step: number;
+  scrollStep: number;
+  label: string;
+  modelValue: number;
+}>();
+
+const inputValue = ref(props.modelValue);
+watchEffect(() => {
+  inputValue.value = props.modelValue;
+});
+const inputValueFixed = computed(() => inputValue.value.toFixed(2));
+
+const emit = defineEmits<{
+  "update:modelValue": [value: number];
```

Comment:
これが呼ばれるタイミングをドキュメント書いてあげた方がいいかもですね！

たぶん設計として、変更中のスライダーの値は親コンポーネントに通知されず、変更が確定したタイミングで通知されるようにされてる感じですよね！
すごくいいと思います！！
この理解があってれば、コメントはこんな感じとか？
```suggestion
  /** 値が確定したタイミングで実行される */
  "update:modelValue": [value: number];
```